docs/reference/modules/asset.md

```diff
@@ -29,7 +29,7 @@ By default, `watch` is set to `true`. A truthy value will cause the application
 
 ### `uploadfs`
 
-This optional property can be used to configure an `uploadfs` instance that differs from the one configured by the `attachment` module, allowing changes in where media is stored and how it is served. Full documentation for uploadfs can be found [here](https://www.npmjs.com/package/uploadfs).
+When the `APOs_UPLOADFS_ASSETS` env variable is present,this optional property can be used to configure an `uploadfs` instance that differs from the one configured by the `attachment` module, allowing changes in where assets from the webpack build process are stored and how they are served. Full documentation for uploadfs can be found [here](https://www.npmjs.com/package/uploadfs).
```

Comment:
* "env" -> "environment" is worth it I think
* ",this" should be ", this" (ultimate nitpicking prize sorry)
* We need to cover `APOS_UPLOADFS_ASSETS` in general, but that belongs in your forthcoming guide to the `@apostrophecms/uploadfs` module, so it's OK that we don't do it here
* `APOs_` should be `APOS_`

Reply:
Gah... tired eyes today are for making dumb mistakes.